doc/classes/AudioEffectCapture.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="AudioEffectCapture" inherits="AudioEffect" version="4.0">
+	<brief_description>
+		Captures audio from an audio bus in real-time.
+	</brief_description>
+	<description>
+		AudioEffectCapture is an AudioEffect which copies all audio frames from the attached audio effect bus into its internal ring buffer.
+		Application code should consume these audio frames from this ring buffer using [method get_buffer] and process it as needed, for example to capture data from a microphone, implement application defined effects, or to transmit audio over the network.
+	</description>
+	<tutorials>
+	</tutorials>
+	<methods>
+		<method name="can_get_buffer" qualifiers="const">
+			<return type="bool">
+			</return>
+			<argument index="0" name="frames" type="int">
+			</argument>
+			<description>
+				Returns [code]true[/code] if at least [code]frames[/code] audio frames are available to read in the internal ring buffer.
+			</description>
+		</method>
+		<method name="clear_buffer">
+			<return type="void">
+			</return>
+			<description>
+				Clears the internal ring buffer.
+			</description>
+		</method>
+		<method name="get_buffer">
+			<return type="PackedVector2Array">
+			</return>
+			<argument index="0" name="frames" type="int">
+				Number of audio frames to return.
```

Comment:
Arguments can't be documented this way, running `doctool` would remove it. Argument-specific details can be given in the `<description>`.